### DISCUSSION TITLE
Enabling Github Discussions on jupyter organization

### DISCUSSION THREAD

**Discussion by ivanov** [jupyter/software-steering-council-team-compass#10] [General]
I'd like to propose enabling Discussions for the Jupyter organization on Github.

One of the use cases could be to allow the Software Steering Council a more [suitable place to conduct asynchronous discussions](https://github.com/jupyter/software-steering-council-team-compass/issues/7#issuecomment-1690553290) than what is possible in regular Github issues (Discussions allow for threading, for example).

It could also be used as a way to direct people to various subprojects (since many of the subprojects have their own github organization, but users would naturally show up at this organization), as well as directing users to the [discourse](https://discourse.jupyter.org/).

cc @jupyter/software-steering-council and @jupyter/executive-council 

![image](https://github.com/jupyter/software-steering-council-team-compass/assets/118211/2ac2756a-e49f-4df3-a829-fd4ee447572d)

I want to open this up for (_wait for it!_) **discussion** (_see what I did there?_) to see what others think.

**Comment by echarles** [comment]
Should we open a Discussion to discuss that idea ? :) Sorry, my day has been hard than expected, just wanted to write something not serious and stupid.

More seriously, I am reducing the channels I monitor and engage into, so personally I prefer not being in those GitHub Discussions (I also don't engage in other repo Discussions). But don't wait on me if you think it brings value.

**Comment by echarles** [comment]
> But don't wait on me if you think it brings value.

... if you  think it brings value compared to the GitHub issues. I always wonder what should be created as a Discussion or as an issue, so I create an issue by default, which has already a lot of bells and whistles to prioritize, filter, search, some of which we don't use yet (like milestones linked to a GitHub project...).

**Comment by ivanov** [comment]
discussions are enabled here now, so closing this.
